toolkits/global/packages/global-expander/js/expander.js

```diff
@@ -9,6 +9,7 @@ const defaultOptions = {
 	TRIGGER_OPEN_CLASS: 'is-open',
 	TRIGGER_OPEN_LABEL: undefined,
 	CLOSE_ON_FOCUS_OUT: true,
+	TARGET_FOCUS: true,
```

Comment:
Does this option make the `AUTOFOCUS` feel a bit confusing?
What happens if they're both set to true?

One focuses on the target element, the other focuses on the first tabbable element _inside_ the target.

Reply:
I'm wondering if you should keep `AUTOFOCUS` but it should take a string instead of a boolean.
Maybe `AUTOFOCUS: 'target'`, `AUTOFOCUS: 'child'`, `AUTOFOCUS: null` (null being the default 🤔 ?